core/src/cluster_slot_state_verifier.rs

```diff
@@ -472,9 +497,6 @@ fn on_dead_slot(slot: Slot, dead_state: DeadState) -> Vec<ResultingStateChange>
         state_changes.push(ResultingStateChange::SendAncestorHashesReplayUpdate(
             AncestorHashesReplayUpdate::Dead(slot),
         ));
-        if is_slot_duplicate {
-            state_changes.push(ResultingStateChange::MarkSlotDuplicate(Hash::default()));
```

Comment:
It seems this doesn't actually do anything, as a dead slot in fork choice won't have any votes or descendants (those will be marked for the correct `SlotHashKey`), so it is safe to remove.

Reply:
yeah exactly